offer-manager-domain/src/main/groovy/life/qbic/business/persons/affiliation/Affiliation.groovy

```diff
@@ -124,15 +124,36 @@ class Affiliation {
         this.category = category
     }
 
+    /**
+     * Getter that is required by Hibernate and other JPA implementing frameworks.
+     * @return the status of the current affiliation
+     */
     protected boolean getIsActive() {
         return isActive
     }
 
+    /**
+     * Requests the current status of the affiliation entity.
+     * <p>
+     * An affiliation that has been archived, is inactive. So this method will return
+     * <code>false</code> in this case. Otherwise the return value is <code>true<code>.
+     * <p>
+     * We need to keep affiliations stored, even if they are not used anymore, to preserve
+     * backwards compatibility for offer generation.
+     * <p>
+     * To make the intention clear, that they should not be used anymore (outdated, wrong, etc),
+     * this flag has been introduced.
+     * @return
+     */
     boolean isActive() {
         return getIsActive()
```

Comment:
I would not accept the suggestion but implement it in the could, the suggestion mode was strange .. 
```suggestion
boolean isArchived() {
        return !getIsActive()
```